packages/ui/resolution/resolution.js

```diff
@@ -18,7 +18,7 @@ let baseScreen,
 	config;
 
 /**
- * Object that stores all of the pixel conversion factors to each keyed unit.
+ * Object that stores all the pixel conversion factors to each keyed unit.
```

Comment:
How about fixing it like this?
```suggestion
 * Object that stores the pixel conversion factors to each keyed unit.
```